_docs-sources/foundations/accounts/manage-accounts.md

```diff
@@ -0,0 +1,24 @@
+# Manage Accounts
+
+This document provides instructions for managing AWS accounts in AWS Control Tower using Gruntwork. Not all account management operations are currently available using the Gruntwork Account Factory, this document provides guidance on which operations _should_, _may_, or _must_ be managed either in the AWS Console or programmatically using Gruntwork tools using Infrastructure as Code (IaC).
```

Comment:
Proposed rewording since this doc doesn't really provide instructions (yet?). I'd think we would either link to instruction pages for each operation below or include the documentation on this page